confgenerator/confgenerator.go

```diff
@@ -395,12 +396,16 @@ func (l *Logging) generateFluentbitComponents(userAgent string, hostInfo *host.I
 
 func generateSeveritySelfLogsParser() []fluentbit.Component {
 	out := make([]fluentbit.Component, 0)
+
+	_, timeOffset := time.Now().Zone()
```

Comment:
`timeOffset` is in seconds here, but `Time_Offset` in fluent-bit is expecting a `+-hhmm`-style offset. Leading zero is also required. Something like this should work:

```go
timeOffset := time.Now().Format("-0700")
```

Reply:
Great catch, thanks!